ambry-account/src/main/java/com/github/ambry/account/mysql/AccountDao.java

```diff
@@ -946,15 +951,15 @@ private int executeUpdateDatasetSqlIfExpiredStatement(PreparedStatement statemen
     } else {
       statement.setObject(2, null);
     }
-    if (userTags != null) {
-      statement.setString(3, userTagsInJson);
+    if (retentionTimeInSeconds != null) {
```

Comment:
Same question here. 
Unless we check which function is calling the function and what the statement passes in, we cannot verify if the statement.setXXX(2, xxx) sets the right parameter.

Reply:
I feel like the integration test is good enough to verify it, besides, we have the local mysql database which can clearly check what we set.